components/Sidebar/SidebarLink.tsx

```diff
@@ -20,50 +21,65 @@ const SidebarLink = ({
   closeDrawer,
 }: {
   menuList: MenuListProps[];
-  closeDrawer: ()=>void;
+  closeDrawer: () => void;
 }) => {
-    return (
-      <Box w="full">
-        <Accordion defaultIndex={[0]} allowToggle>
-          {menuList.map((menu,key) => (
-              <AccordionItem key={ key}>
-              <h2>
-                <AccordionButton>
-                  <Box flex="1" textAlign="left">
-                    {menu.category}
-                  </Box>
-                  <AccordionIcon />
-                </AccordionButton>
-              </h2>
-              <AccordionPanel pb={4}>
-                {menu.blogs.map((blog, key) => (
-                  <Box key={key} cursor={"pointer"} onClick={closeDrawer}>
-                    <Link href={`/blog/${blog.pageId}`}>
-                      <Flex
-                        fontSize="14px"
-                        color="#37352F"
-                        key={blog.pageId}
-                        fontFamily="Segoe UI"
-                        fontWeight={"600"}
-                        px="2"
-                        borderRadius={"2"}
-                        py="1"
-                        _hover={{
-                          bg: "#E2E2E1",
-                        }}
-                      >
-                        <Text px="1">{blog.icon}</Text>
-                        <Text>{blog.title}</Text>
-                      </Flex>
-                    </Link>
-                  </Box>
-                ))}
-              </AccordionPanel>
-            </AccordionItem>
-          ))}
-        </Accordion>
-      </Box>
-    );
+  const router = useRouter();
+  const { slug } = router.query;
+  const [activeAccordion, setActiveAccordion] = React.useState(-1);
```

Comment:
I don't see this being used anywhere. You are using a different method to check for an active link, right?

Reply:
I am using slug to get active tab

Reply:
yeah but the tab is not open, can you make it so that it is open when a category has an active link?